gst/nnstreamer/tensor_transform/tensor_transform.h

```diff
@@ -35,6 +35,7 @@
 #include <gst/gst.h>
 #include <gst/base/gstbasetransform.h>
 #include <tensor_common.h>
+#include <tensor_data.h>
```

Comment:
Do we export this header?

Reply:
export for devel pkg? no, this header file is from different include-path in build script.

Reply:
IMHO, although the directory where this header is placed is provided with ```-I``` option, it is better to directly indicate the user header path for better compile performance (maybe old-school).

```c
#include "../tensor_data.h"
```

Whatever you choose, LGTM here.